app/app.go

```diff
@@ -39,7 +39,7 @@ var appConfig = configDefaults{
 	Name:     os.Args[0],
 	HelpName: os.Args[0],
 	Usage:    "A Brooklyn command line client application",
-	Version:  "0.9.0",
+	Version:  "0.10.0-SNAPSHOT",
```

Comment:
Add an inline comment `BROOKLYN_VERSION`, or a comment on the line above `BROOKLYN_VERSION_BELOW` to have this changed automatically.